app/components/edit/tab_form/tab_component.rb

```diff
@@ -36,7 +36,7 @@ def selected?
       end
 
       def classes
-        merge_classes('nav-link', selected? ? 'active' : nil)
+        merge_classes('nav-link w-100', selected? ? 'active' : nil)
```

Comment:
Never place form tabs beside one another. Keep them as a stack of vertical pills.